### PR TITLE
Remove deprecated stuff

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,3 +6,6 @@ Migration Instructions
 
 * The `D_GC` variable was removed as it wasn't used internally anymore.
   If you made use of it anywhere, you should define it yourself (in `Config.mak` for example) from now on.
+
+* The deprecated packaging function `FUN.mapbin()` was removed, use
```
Comment (gkotian) [inline RELEASE_NOTES.md]: `s/mapbin/mapbins`

**Comment by leandro-lucarella-sociomantic** [comment]
Updated.

**Review by gkotian — APPROVED** [review]
LGTM.

**Comment by leandro-lucarella-sociomantic** [comment]
@nemanja-boric-sociomantic ?

**Pull request merged**